volatility/framework/symbols/intermed.py

```diff
@@ -369,7 +369,7 @@ def _interdict_to_template(self, dictionary: Dict[str, Any]) -> interfaces.objec
             elif type_name == 'enum':
                 update = self._lookup_enum(dictionary['name'])
             elif type_name == 'bitfield':
-                update = {'start_bit': dictionary['bit_position'], 'end_bit': dictionary['bit_length']}
+                update = {'start_bit': dictionary['bit_position'], 'end_bit': dictionary['bit_position'] + dictionary['bit_length']}
```

Comment:
Great spot!  5:)
It looks like the only place we use [`end_bit`](https://github.com/volatilityfoundation/volatility3/blob/master/volatility/framework/objects/__init__.py#L370), we use it as the actual end rather than the length, so this makes sense.